README.md

```diff
@@ -0,0 +1,166 @@
+- Profile avatars
+- Text avatars
+- Square avatars
+
+You can check the `avatar` component here : https://linearui.netlify.app/pages/components/avatar/avatar
+
+
+---
+
+### Badges
+
+Badges are used to highlight user's status or a number count of an item. Linear UI has following types of badges :
+- Badge on avatars
+- Badge on icons
+
+You can check the `badge` component here : https://linearui.netlify.app/pages/components/badge/badge
+
+---
+
+### Buttons
+
+Buttons allow users to make choices and take action by a single tap. Linear UI has following types of buttons:
+- Solid buttons
+- Outline buttons
+- Link buttons
+- Icon buttons
+- Floating action buttons
+
+You can check the `button` component here : https://linearui.netlify.app/pages/components/buttons/button
+
+---
+
+### Cards
+
+Card components is used to group a set of information into a container which can be used throughout your website. Linear UI has following types of buttons: 
+- Basic cards
+- Vertical cards
+- Horizontal cards
+- Text only cards
+- Card with text overlay
+- Card with a badge
+- Card with a dismiss button
+
+You can check the `card` component here : https://linearui.netlify.app/pages/components/cards/cards
+
+---
+
+### Images
+
+The Image component allows you to place an image on your page. Linear UI has following types of images :
+- Responsive images
+- Round images
+- square images
+
+You can check more about the `image` component here : https://linearui.netlify.app/pages/components/image/image
+
+---
+
+### Inputs
+
+The Input component is used to create interactive controls for web-based forms in order to accept data from the user. Linear UI has following types of inputs :
+- Standard Inputs
+- Validation Inputs
+- Radio Inputs
+- Checkbox Inputs
+
+You can check more about the `input` component here : https://linearui.netlify.app/pages/components/input/input
+
+---
+
+### Modal
+
+A modal is a web page element that displays in front of and deactivates all other page content. Currently Linear UI has only one type of modal.
+
+You can check the `modal` component here: https://linearui.netlify.app/pages/components/modal/modal
+
+---
+
+### Typography
+
+Typography describes the creative design of the text on your web pages. Linear UI has following types of typography styles:
+- Headings
+- Paragraph texts
+- Alligned texts
+- Some other type of text styles
+   - strikethrough
+   - bold
+   - light weight 
+   - gray text
+
+You can check the `typography` component here : https://linearui.netlify.app/pages/components/typography/typography
+
+---
+
+### Toast
+
+Toasts are intended to provide feedback to a user and self-dismiss after a short time with an option to self-dismiss. Linear UI has following type of toasts :
+- Simple Toasts
+- Toasts with buttons
+
+You can check more about the `toasts` here : https://linearui.netlify.app/pages/components/toast/toast
+
+---
+
+Thanks for reading, hope you like Linear UI.
+
```

Comment:
remove these blank spaces and you are good to merge this branch

Reply:
done